packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -478,7 +493,8 @@ export class GutenbergEditorPage {
 		const frame = await this.getEditorFrame();
 		await frame.click( selectors.previewButton );
 		await frame.click( selectors.previewMenuItem( target ) );
-		await frame.waitForSelector( selectors.previewPane( target ) );
+		const handle = await frame.waitForSelector( selectors.previewPane( target ) );
+		await handle.waitForElementState( 'stable' );
```

Comment:
The viewport animates when a preview device is selected (eg. Desktop -> Mobile).